data/schema/v2/Decision_Point_Value_Selection-2-0-0.schema.json

```
@@ -0,0 +1,226 @@
+  "type": "object",
+  "properties": {
+    "timestamp": {
+      "description": "Timestamp of the selections, in RFC 3339 format.",
+      "examples": [
+        "2025-01-01T12:00:00Z",
+        "2025-01-02T15:30:45-04:00"
+      ],
+      "format": "date-time",
+      "title": "Timestamp",
+      "type": "string"
+    },
+    "schemaVersion": {
+      "const": "2.0.0",
+      "description": "The schema version of this selection list.",
+      "title": "Schemaversion",
+      "type": "string"
+    },
+    "target_ids": {
+      "description": "Optional list of identifiers for the item or items (vulnerabilities, reports, advisories, systems, assets, etc.) being evaluated by these selections.",
+      "examples": [
+        [
+          "CVE-1900-0000"
+        ],
+        [
+          "VU#999999",
+          "GHSA-0123-4567-89ab"
+        ]
+      ],
+      "items": {
+        "type": "string"
+      },
+      "minItems": 1,
+      "title": "Target Ids",
+      "type": "array"
+    },
+    "selections": {
+      "description": "List of selections made from decision points. Each selection item corresponds to value keys contained in a specific decision point identified by its namespace, key, and version. Note that selection objects are deliberately minimal objects and do not contain the full decision point details.",
+      "items": {
+        "$ref": "#/$defs/Selection"
+      },
+      "minItems": 1,
+      "title": "Selections",
+      "type": "array"
+    },
+    "resources": {
+      "description": "A list of references to resources that provide additional context about the decision points found in this selection.",
+      "examples": [
+        [
+          {
+            "description": "Documentation for a set of decision points",
+            "uri": "https://example.com/decision_points"
+          },
+          {
+            "description": "JSON representation of decision point 2",
+            "uri": "https://example.org/definitions/dp2.json"
+          },
+          {
+            "description": "A JSON file containing extension decision points in the x_com.example namespace",
+            "uri": "https://example.com/ssvc/x_com.example/decision_points.json"
+          }
+        ]
+      ],
+      "items": {
+        "$ref": "#/$defs/Reference"
+      },
+      "minItems": 1,
+      "title": "Resources",
+      "type": "array"
+    },
+    "references": {
+      "description": "A list of references to resources that provide additional context about the specific values selected.",
+      "examples": [
+        [
+          {
+            "description": "A report on which the selections were based",
+            "uri": "https://example.com/report"
+          }
+        ]
```

Let's add another example here that points to a specific line in code:
```suggestion
        ],
        [
          {
            "description": "A code section on which the selections were based",
            "uri": "https://git.example.com/some-relevant-path/code#L21-42"
          },
          {
            "description": "A code section on which calls the vulnerable function",
            "uri": "https://git.example.com/some-relevant-path/callingcode#L91-16"
          }
        ]
```